src/merkle/smt/simple/mod.rs

```diff
@@ -0,0 +1,325 @@
+use crate::{
+    merkle::{EmptySubtreeRoots, InnerNodeInfo, MerkleTreeDelta, StoreNode, ValuePath},
+    utils::collections::TryApplyDiff,
+    EMPTY_WORD,
+};
+
+use super::{
+    InnerNode, LeafIndex, MerkleError, NodeIndex, RpoDigest, SparseMerkleTree, Word, SMT_MAX_DEPTH,
+    SMT_MIN_DEPTH,
+};
+use crate::utils::collections::{BTreeMap, BTreeSet};
+
+#[cfg(test)]
+mod tests;
+
+// SPARSE MERKLE TREE
+// ================================================================================================
+
+/// A sparse Merkle tree with 64-bit keys and 4-element leaf values, without compaction.
+///
+/// The root of the tree is recomputed on each new leaf update.
+#[derive(Debug, Clone, PartialEq, Eq)]
+#[cfg_attr(feature = "serde", derive(serde::Deserialize, serde::Serialize))]
+pub struct SimpleSmt<const DEPTH: u8> {
```

Comment:
This broke downstream, do we have PRs to fix it on the works?

Reply:
Yes - all fixing PRs have been merged. If you rebase from `main` and do `cargo update` it should fix the node build.